llvm/lib/Target/X86/GISel/X86CallLowering.cpp

```diff
@@ -218,14 +218,14 @@ struct X86IncomingValueHandler : public CallLowering::IncomingValueHandler {
 
   void assignValueToReg(Register ValVReg, Register PhysReg,
                         const CCValAssign &VA) override {
-    markPhysRegUsed(PhysReg);
+    markPhysRegUsed(PhysReg.asMCReg());
```

Comment:
I'm using asMCReg() here to assert that PhysReg is really a physical register instead of silently implicitly converting to MCRegister. `addLiveIn` used by one of the overrides of `markPhysRegUsed` uses `MCRegister` so I converted early.

Reply:
Just finished checking, no unexpected failures on benchmarks.